src/handlers/fetch-api/platform-adapter.ts

```diff
@@ -0,0 +1,39 @@
+import {
+  HandlerUseCaseResult,
+  PlatformAdapterInterface,
+} from "../shared/generic-adapter-use-case-types";
+
+export type WebApiHandlerInput = Request;
+export type WebApiHandler = (req: Request) => Response | Promise<Response>;
+
+export class WebApiAdapter implements PlatformAdapterInterface<WebApiHandlerInput> {
```

Comment:
add alias (in package json) to provide a separate path like `handlers/next-app-router` for better DX